typha/pkg/config/config_params.go

```diff
@@ -121,7 +121,8 @@ type Config struct {
 	ServerPingIntervalSecs               time.Duration `config:"seconds;10"`
 	ServerPongTimeoutSecs                time.Duration `config:"seconds;60"`
 	ServerHandshakeTimeoutSecs           time.Duration `config:"seconds;10"`
-	ServerPort                           int           `config:"port;0"`
+	ServerPort                           int           `config:"int(0,65535);5473"`
+	ServerHost                           string        `config:"host-address;"`
```

Comment:
Do we need / want to explicitly default this to `0.0.0.0` for backwards compatibility?

Reply:
Hmm I tested it in playground and it listens  on `[::]:<port>` which also listens on ipv4 when the kernel is configured for dual-stack.

ref: https://serverfault.com/a/39561